message_ix_models/util/scenarioinfo.py

```diff
@@ -87,7 +89,7 @@ class ScenarioInfo:
     #: :obj:`True` if a MESSAGE-MACRO scenario.
     is_message_macro: bool = False
 
-    _yv_ya: pd.DataFrame = None
+    _yv_ya: pd.DataFrame | None = None
```

Comment:
Note this is only valid from Python 3.10. To keep compatibility with 3.9 and 3.8, we need to use `Optional[pd.DataFrame]`.